src/lib/expansion/expansion-panel-header.ts

```diff
@@ -140,8 +140,11 @@ export class MatExpansionPanelHeader implements OnDestroy, FocusableOption {
       // Toggle for space and enter keys.
       case SPACE:
       case ENTER:
-        event.preventDefault();
-        this._toggle();
+        if (!event.altKey && !event.metaKey && !event.shiftKey && !event.ctrlKey) {
```

Comment:
Not in this PR, but we should think about adding some helper functions to cdk/keycodes for checking stuff like this